content/download/debian.adoc

```diff
@@ -6,63 +6,194 @@ weight = 10
 :icons: fonts
 :iconsdir: /img/icons/
 
-== Debian Stable (Stretch)
+== Basic Information
 
-{{< repology debian_stable_backports >}}
+The information further down about the available KiCad packages might not
+always be fully up2date and correct due ongoing development. For getting an
+overview about the current most recent version for the KiCad packages within
+the various Debian releases please have a look at
+https://packages.debian.org/search?lang=en&keywords=kicad[package.debian.org].
 
-{{< repology debian_stable >}}
+Please note also that Debian stable releases don't get updates for existing
+packages except security updates. Packages within the Stable releases get
+usually newer and recent versions through backports repositories. KiCad is
+here no special case.
 
-The release 4.0.5 is available for Debian
-https://packages.debian.org/stretch/kicad[stable/stretch], the backported
-version 5.0.1 is available in
-https://packages.debian.org/stretch-backports/kicad[stretch-backports].
+To be able to install versions from a official Debian Backport repository you
+need to extend your sources.list configuration if not yet happen. If you are
+running Debian Buster please create a file
+`/etc/apt/sources.lists.d/buster-backports.list` if you don't have an similar
+entry already.
 
-You can install it with the following commands in a terminal, otherwise you can
-use any of the package managers you like:
+[source,bash]
+----
+# /etc/apt/sources.lists.d/buster-backports.list
+deb http://deb.debian.org/debian buster-backports main contrib non-free
+----
+
+If your system is still running Debian Stretch create a similar file (if not
+done somehow already or created by the installer similar)
+`/etc/apt/sources.lists.d/stretch-backports.list`
+like above except the content is pointing to stretch-backports.
+
+[source,bash]
+----
+# /etc/apt/sources.lists.d/stretch-backports.list
+deb http://deb.debian.org/debian buster-backports main contrib non-free
```

Comment:
shouldn'y this be stretch-backports?

Reply:
Yes, of course, damned c&p error. Changed within next force-push.